rsmtool/reader.py

```diff
@@ -22,6 +22,47 @@
 from rsmtool.container import DataContainer
 
 
+
+def read_jsonlines(filename, converters=None):
+    """ Read jsonlines from a file. 
+    Normalize nested jsons with up to one level of nesting
+
+    Parameters
+    ----------
+    filename: str
+        Name of file to read
+    converters : dict or None, optional
+        A dictionary specifying how the types of the columns
+        in the file should be converted. Specified in the same
+        format as for `pd.read_csv()`.
+
+    Returns
+    -------
+    df : pandas DataFrame
+         Data frame containing the data in the given file.
+    """
+
+    df = pd.read_json(filename,
+                      orient='records',
+                      lines=True,
+                      dtype=converters)
+        
+    # let's check if we have nested columns
+    nested_cols = [c for c in df.columns if isinstance(df[c][0],dict)]
+
+    if len(nested_cols) > 0:
+        dfs = []
+        for col in nested_cols:
+            df_nested_column = pd.io.json.json_normalize(df[col])
+            dfs.append(df_nested_column)
+            df.drop(col, axis=1, inplace=True)
+        df_new = pd.concat([df] + dfs, axis=1)
```

Comment:
If we just assign this back to `df`, can we get rid of the `else` below?

Reply:
Not sure - I think different var names make the code more readable but it's probably a judgement call. @desilinguist ?